fiftyone/server/samples.py

```diff
@@ -22,6 +22,16 @@
 from fiftyone.server.utils import from_dict
 import fiftyone.server.view as fosv
 
+import logging
+
+logging.basicConfig(
+    filename="/tmp/fiftyone.log",
+    filemode="a",
+    format="%(asctime)s,%(msecs)d %(name)s %(levelname)s %(message)s",
+    datefmt="%H:%M:%S",
+    level=logging.INFO,
+)
```

Comment:
Is this for debugging? I'm not sure this should be used in production. Should probably discuss with other backend folks

Reply:
ah, this is just for debugging, didn't mean to commit.